src/ws/app.rs

```diff
@@ -227,10 +244,20 @@ async fn send_update_to_corresponding_generator(
         RwLock<HashMap<Address, GeneratorYielder<SubscribeFriendshipEventsUpdatesResponse>>>,
     >,
     event_update: Event,
+    metrics: Arc<Metrics>,
 ) {
     if let Some(response) = event_as_friendship_update_response(event_update.clone()) {
         let corresponding_user_id = Address(event_update.to.to_lowercase());
 
+        record_procedure_call_and_duration_and_size(
+            metrics,
+            None,
+            Procedure::SubscribeFriendshipEventsUpdates,
+            Instant::now(),
+            response.encoded_len(),
```

Comment:
This is when a user sends a friendship event, I'm not sure what to do in this case:
a. Should we track this as a request of the method of send update? Or are we measuring it twice?
b. Should we track this as a response when the user reads the generator?
c. Both?